src/pages/fuksis/fuksiguide.tsx

```diff
@@ -6,12 +6,12 @@ import PageLayout from "@src/components/layouts/PageLayout"
 
 const Fuksiguide: React.FC<PageProps> = ({ pageContext }) => {
   return (
-    <PageLayout pageContext={pageContext} title="Fuksi 2023">
-      <div className={style.text}>Fuksiopas sisältää kaiken, mitä sinun tarvitsee tietää tulevasta opiskelijaelämästä ja AS-killasta.</div>
-
-      <a className={style.rounded + " rounded-button"} target="_blank" href="https://drive.google.com/file/d/14Ew0gsN5HKbIRWwXVne50ICVLe5d35TC/view?usp=sharing">Lataa PDF</a>
-
-      <div >
+    <PageLayout pageContext={pageContext} title="Fuksi 2023" lead="Fuksiopas sisältää kaiken, mitä sinun tarvitsee tietää tulevasta opiskelijaelämästä ja AS-killasta.">
+      <div style={{width: "100%"}}>
+        <a className={style.rounded + " rounded-button"}
```

Comment:
style.rounded was deleted. Could bring back to give the button some margins so it doesn't touch the hero.